azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/boot/initializer/SpringBootTelemetryInitializer.java

```diff
@@ -23,9 +23,8 @@
 
   @Override
   public void initialize(Telemetry telemetry) {
-
-    DeviceContext device = telemetry.getContext().getDevice();
-    device.setRoleName(appName);
+    CloudContext device = telemetry.getContext().getCloud();
```

Comment:
well device variable name doesn't make sense now. may be something related to cloud as name ?:)

Reply:
correct. I'll fix that

Reply:
[ ] testing checkbox...